index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>Product Landing Page</title>
+        <link rel="stylesheet" href="style.css">
+    </head>
+
+    <body>
+        <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+        <header id="header">
+            <img src="logo.png" alt="" id="header-img">
+            <nav id="nav-bar">
+                <a href="#header" class="nav-link">Home</a>
+                <a href="#about" class="nav-link">About</a>
+                <a href="#contact" class="nav-link">Contact</a>
```

Comment:
Please consider using <code>ul</code> with a class of **links** and <code>li</code> to enclose each link with a class of **link-item** it more meaningful.